io/serialport/25-serial.js

```diff
@@ -200,6 +224,42 @@ module.exports = function(RED) {
     }
     RED.nodes.registerType("serial request", SerialRequestNode);
 
+
+    // Serial Control Node
+    function PortSelectNode(n) {
+        RED.nodes.createNode(this,n);
+        this.serialConfig = RED.nodes.getNode(n.serial);
+
+        if (!this.serialConfig) {
+            this.error(RED._("serial.errors.missing-conf"), {});
+            return;
+        }
+
+        this.serial = n.serial;
+        var node = this;
+        node.port = serialPool.get(this.serialConfig);
+        node.on("input",function(msg) {
+            if (msg.payload.hasOwnProperty("config")) { 
+                if (msg.payload.config === "query") { 
+                    node.send({payload: node.serialConfig});
+                } else {
+                    node.serialConfig.changePort(msg.payload.config);
+                    node.serialConfig.emit('start',msg.payload.config);
+                    node.send({payload: msg.payload.config});
+                }
+            } else if (msg.payload.hasOwnProperty("start")) { 
```

Comment:
Rather than two separate properties - I'd prefer a single "enable" (or control ?)  property that is set to true or false and maybe also allows start and stop and enable and disable ?